app/src/organisms/ModuleWizardFlows/Success.tsx

```diff
@@ -24,32 +32,83 @@ export const BODY_STYLE = css`
   }
 `
 
-export const Success = (
-  props: ModuleSetupWizardStepProps
-): JSX.Element | null => {
-  const { proceed, attachedModule, isRobotMoving, isOnDevice } = props
+interface SuccessProps extends ModuleSetupWizardStepProps {
+  setSelectedModule: (module: AttachedModule | null) => void
+}
+
+export const Success = (props: SuccessProps): JSX.Element | null => {
```

Comment:
```suggestion
export function Success(props: SuccessProps): JSX.Element  {
```
Nit, I know you didn't write this line, but I think we should probably strive to keep components with the `function foo()` syntax. Some discussion on this [here](https://opentrons.slack.com/archives/C6PF2M2LC/p1742318486373119?thread_ts=1742316055.038139&cid=C6PF2M2LC) if you're interested.

What's more important (and again, not your code) is this component doesn't return `null`, nor should any component return `null`.